test/puppetlabs/trapperkeeper/services/status/status_service_test.clj

```diff
@@ -219,3 +240,83 @@
                     :y "y"
                     :z "y"}
                   (response->status resp)))))))))
+
+(deftest proxy-ssl-status-endpoint-test
+  (testing "status-proxy-service can connect to https status-service service correctly"
+    ; Start status service
+    (with-app-with-config
+      status-app
+      [jetty9-service/jetty9-service
+       webrouting-service/webrouting-service
+       status-service
+       foo-service]
+      ssl-status-service-config
+      ; Start the proxy service
+      (with-app-with-config
+        proxy-app
+        [jetty9-service/jetty9-service
+         webrouting-service/webrouting-service
+         status-proxy-service]
+        status-proxy-service-config
+        ; Make HTTP request to the proxy, which will forward it as an HTTPS request
+        (let [resp (http-client/get "http://localhost:8181/status-proxy/v1/services")
```

Comment:
I think we should also querying for a specific service to ensure that that is being proxied.

Might also be worth doing a simple test to ensure that the `level` or `service_status_version` query params work with the proxy service.